src/server/models/users.model.js

```diff
@@ -4,22 +4,32 @@ import mongoose from 'mongoose';
 
 const Schema = mongoose.Schema;
 let userSchema = new Schema({
-  userId: String,
-  email: String,
-  name: String,
-  age: Number,
-  field: String,
-  region: String,
-  skills: String,
-  regDate: {
-    type: Date,
-    default: Date.now(),
-  },
+    userId: String,
+    email:
+    { type: String,
+        unique: true,
+        required: true
+    },
+    platform_id: String,
```

Comment:
Can we make platform_id to be an enum so we first start with linkedIn / FB and later add more types?